code/__DEFINES/atmospherics.dm

```diff
@@ -168,13 +168,13 @@
 
 //TANKS
 /// temperature in kelvins at which a tank will start to melt
-#define TANK_MELT_TEMPERATURE 1000000
+#define TANK_MELT_TEMPERATURE 1000 + T0C
 /// Tank starts leaking
-#define TANK_LEAK_PRESSURE (30.*ONE_ATMOSPHERE)
+#define TANK_LEAK_PRESSURE (20 * ONE_ATMOSPHERE + 5)
```

Comment:
This is higher than the rupture pressure so tanks probably won't leak (for long) they'll just pop

Reply:
I removed tank leaking from internal tanks, it didn't seem to even work as intended and during testing it was like... meh honestly